guides/v2.3/graphql/develop/resolvers.md

```diff
@@ -68,6 +68,9 @@ Syntax option | Description
 `@resolver(class)` | The class of the resolver
 `@doc(description)` | Describes the purpose of the mutation
 
+{:.bs-callout .bs-callout-tip}
```

Comment:
I would extend it with a simple example. I.e

Wrong approach:
```graphql
type Mutation {
    mutationQueryName(param1: String, param2: Int, ...): MutationQueryOutput @resolver(class: "Magento\\<module_name>\\Model\\Resolver\\MutationResolverModel") @doc(description:"Mutation query description")
}
```

Correct approach:
```graphql
type Mutation {
    mutationQueryName(inputParam: InputParamsType): MutationQueryOutput @resolver(class: "Magento\\<module_name>\\Model\\Resolver\\MutationResolverModel") @doc(description:"Mutation query description")
}

type InputParamsType {
    param1: String
    param2: Int
}
```

Thank you.

Reply:
**added**

Thank you, @rogyar 👍

Reply:
I like the examples. Thanks. 

I'm checking with the internal GraphQL team, since we have mutations that don't follow this advice.

`generateCustomerToken(email: String! password: String!): CustomerToken`

Reply:
This is a proposition regards to BEST PRACTICES ;)

Many thanks, @keharper !

Reply:
@keharper , did you have a chance to verify changes in this PR?

Thank you!